### DISCUSSION TITLE
Which users viewed the content

### DISCUSSION THREAD

**Discussion by jledesma-opsguru** [mediacms-io/mediacms#730] [Q&A]
Is it possible to see which users viewed a specific video and not just the number of views?

**Comment by mgogoulos** [comment]
It's not possible now but this information is getting logged so it could be added with not that much effort I believe.

**Reply by jledesma-opsguru** [reply]
Is that information logged in the DB?

**Reply by mgogoulos** [reply]
yes, it's under table MediaAction
